### DISCUSSION TITLE
Add system for letting customers set a password of an account is created from a webhook

### DISCUSSION THREAD

**Discussion by DanielDarrenJones** [statamic-rad-pack/shopify#228] [Feature Requests]
It would be really helpful to have an inbuilt mechanism to allow customers to be able to set a password if they have created an account through the Shopify checkout process.

My current train of thought is sending something similar to a password reset but seperate from the normal reset email, with the ability to export and customise this. That way it can be branded and have the message tweaked. While ensuring the password is secure.

**Comment by ryanmitchell** [comment]
You could listen for the UserCreated event, and if a shopify_id is present trigger your own logic?
